types/nodegit/tag.d.ts

```diff
@@ -10,7 +10,7 @@ export class Tag {
     static createLightweight(repo: Repository, tagName: string, target: Object, force: number): Promise<Oid>;
     static delete(repo: Repository, tagName: string): Promise<number>;
     static list(repo: Repository): Promise<any[]>;
-    static listMatch(tagNames: Strarray | string | string[], pattern: string, repo: Repository): number;
+    static listMatch(pattern: string, repo: Repository): Promise<any[]>;
```

Comment:
Can you make this return an array? As per spec?

Reply:
I believe it returns Promise of array because it is an async function

Reply:
approved